packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -2231,6 +2231,10 @@ export class SnapController extends BaseController<
         this.#calculatePermissionsChange(snapId, processedPermissions);
 
       this.#updateApproval(pendingApproval.id, {
+        ...(manifest.initialConnections !== undefined &&
+        manifest.initialConnections !== null
+          ? { connections: manifest.initialConnections }
+          : {}),
```

Comment:
```suggestion
        initialConnections: snap.manifest?? {}
```
Wouldn't it be simpler to do something like this?

Reply:
My intention here was to omit `initialConnections` completely if it's `undefied` or `null`. I'm not sure if leaving an empty object `{}` makes sense or helps in the process? 🤔 
Also, a lot of tests would require update if I leave the empty object, it would be added everytime, everywhere related to the install/update process. WDYT?

Reply:
It feels like it would be simpler to read and understand if we just always pass it. We do the same for permissions when updating as those may not always be defined either.

I know we would have to update a bunch of tests, but I think it may be worth it.

Reply:
Alright, added it now as you suggested with `?? {}` and updated the tests.